sys/dbgpin/dbgpin.c

```diff
@@ -0,0 +1,18 @@
+
+
+#include "dbgpin.h"
+
+const gpio_t dbgpin_pins[] = { DBGPIN_PINS };
+
+unsigned dbgpin_cnt(void)
+{
+    return ARRAY_SIZE(dbgpin_pins);
+}
+
+void dbgpin_init(void)
+{
+    for (unsigned i = 0; i < ARRAY_SIZE(dbgpin_pins); i++) {
+        gpio_init(dbgpin_pins[i], GPIO_OUT);
+        gpio_clear(dbgpin_pins[i]);
+    }
+}
```

Comment:
I'd also prefer `static inline` here, as there is at most one place it is ever called, getting rid of the function call overhead will certainly improve code size.

Reply:
IMHO premature optimization. Also same problem with `ARRAY_SIZE(dbgpin_pins)`